### PR TITLE
feat: add comparison parameters panel

### PR DESCRIPTION
**Pull request by kirillovzah**

### CODE REVIEW

File: src/components/comparison-params-panel/comparison-params-panel.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
+import { useState } from "react";
+import styled from "styled-components";
+import { getCurrentLayoutProperty, useAppLayout } from "../../utils/layout";
+import { Theme } from "../../types";
+import { ToggleSwitch } from "../toogle-switch/toggle-switch";
+
+type ComparisonParamsProps = {
```
Comment (belom88) [inline src/components/comparison-params-panel/comparison-params-panel.tsx:7]: I suggest moving props typing just before the component.

File: src/components/comparison-params-panel/comparison-params-panel.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  id: string;
+  onClose: () => void;
+};
+
+type LayoutProps = {
+  layout: string;
+};
+
+const Container = styled.div<LayoutProps>`
```
Comment (belom88) [inline src/components/comparison-params-panel/comparison-params-panel.tsx:16]: You have a lot of copy-pasted components here. Create `comparison` folder. Put reusable components in `common.tsx` file. Use components in both panels

**Review by belom88 — CHANGES_REQUESTED** [review]
As we discussed. We start writing unit tests from this sprint for new components. Use as example `input-text.spec.tsx` or other unit test for component.

File: src/components/comparison/common.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  justify-content: ${(props) =>
+    props.panel === Panels.Layers ? "center" : "flex-start"};
+  background: transparent;
+  position: relative;
+  border-radius: 8px;
+  margin-top: 20px;
+  gap: 32px;
+`;
+
+export const CloseButton = styled.div`
```
Comment (belom88) [inline src/components/comparison/common.tsx:43]: I made reusable `CloseButton` in https://github.com/visgl/loaders.gl-showcases/pull/101 . You need to await it to reuse it here

**Review by belom88 — CHANGES_REQUESTED** [review]

File: src/components/comparison/common.tsx (hunk 3 above)
Comment (kirillovzah) [reply src/components/comparison/common.tsx:43]: Ok

**Pull request merged**